kuzzle/kuzzle.go

```diff
@@ -266,5 +339,14 @@ func (k *Kuzzle) SetVolatile(v types.VolatileData) {
 }
 
 func (k *Kuzzle) EmitEvent(e int, arg interface{}) {
-	k.socket.EmitEvent(e, arg)
+	for c := range k.eventListeners[e] {
+		json, _ := json.Marshal(arg)
```

Comment:
Shouldn't we  marshal `arg` argument just once at the beginning of the function, instead of at each loop turn?

Reply:
Well spotted :)